docs/specs/epic1/story12.md

```
@@ -1,143 +1,257 @@
-# Story 1.12: Dockerfile pour Backend Spring Boot
+Parfait, merci pour ces précisions \! Je vais mettre à jour la story et le `Dockerfile` avec ces informations.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)